### DISCUSSION TITLE
Damerau Levenshtein comparison weight taking precedence over exact match weight for exact values

### DISCUSSION THREAD

**Discussion by tombetinashe** [moj-analytical-services/splink#1969] [Q&A]
![image](https://github.com/moj-analytical-services/splink/assets/99027458/2290b617-4868-4007-9c9e-83c22a23b020)
I am currently using splink for record linkage and have the above comparison levels and weights in my model.I have picked that for some records that have exact match on surname/firstname the 6.69 (Damerau Levenshtein comparison weight) is taking precedence over the Exact match weight of 7.75 .I was under the impression that the exact match comparison would take precedence then records that do not fall within that comparison level would be allocated to the rest of the comparison levels.Your assistance would be greatly appreciated.

**Comment by RobinL** [comment]
You're right that with your setup the exact match should take priority

Could it be that some of your values have spaces before/after?

For example, "robin" vs "robin "  (note the trailing space) in which can you'd want to apply data cleaning (trim) prior to importing the data into Splink

If that's not the cause, are you able to provide a reproducible example of the problem?

**Reply by tombetinashe** [reply]
Thank you for the response Robin,the data was cleaned  prior to the linkage but let me just  double check if that is not the problem with the specific records.
